src/cio_websocket.h

```diff
@@ -85,14 +85,6 @@ enum cio_websocket_frame_type {
 	CIO_WEBSOCKET_PONG_FRAME = 0x0a,
 };
 
-
-// Remove and use callbacks or document
-enum cio_websocket_status {
-	CIO_WEBSOCKET_STATUS_OK = 0,
-	CIO_WEBSOCKET_STATUS_CLOSED = -1
-};
-
-
 #define CIO_WEBSOCKET_SMALL_FRAME_SIZE 125
 
 struct cio_websocket {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this structure so it has no more than 20 fields, rather than the 29 it currently has. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=c%3AS1820)